scheduler/src/cook/kubernetes/api.clj

```diff
@@ -407,16 +461,63 @@
     ; "Guaranteed" QoS which requires limits for both memory and cpu.
     (.putLimitsItem resources "cpu" (double->quantity cpus))
     (.setResources container resources)
-    (.setVolumeMounts container (into [] (conj volume-mounts workdir-volume-mount)))
+    (.setVolumeMounts container (into [] (remove nil? (conj volume-mounts
+                                                            (workdir-volume-mount-fn false)
+                                                            (wrapper-script-workdir-volume-mount-fn true)
+                                                            (fileserver-workdir-volume-mount-fn true)))))
     (.setWorkingDir container workdir)
 
     ; pod-spec
     (.addContainersItem pod-spec container)
+
+    ; init container
+    (when-let [{:keys [command image]} init-container]
+      (let [container (V1Container.)]
+        ; container
+        (.setName container cook-init-container-name)
+        (.setImage container image)
+        (.setCommand container ["/bin/sh" "-c" command])
```

Comment:
Do we default to `/bin/sh` because that's also the default shell currently used on Mesos?

Reply:
will verify

Reply:
Cook Executor is using /bin/sh, so yes, this is good.